tools/PI/DevHome.PI/BarWindowVertical.xaml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="utf-8"?>
+<winex:WindowEx
+    x:Class="DevHome.PI.BarWindowVertical"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:winex="using:WinUIEx"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:controls="using:DevHome.PI.Controls"
+    xmlns:helpers="using:DevHome.PI.Helpers"
+    mc:Ignorable="d"
+    Title="" MinHeight="700" MinWidth="70" MaxWidth="70" Width="70" Height="700"
+    TaskBarIcon="Images/pi.ico" IsTitleBarVisible="False"
+    Closed="WindowEx_Closed" >
+
+    <!-- TODO Make DesktopAcrylicBackdrop/MicaBackdrop and AcrylicBackgroundFillColorBaseBrush user-configurable.-->
```

Comment:
> <!-- TODO Make DesktopAcrylicBackdrop/MicaBackdrop and AcrylicBackgroundFillColorBaseBrush user-configurable.--> [](http://example.com/codeflow?start=3&length=113)

nit: out of date comment

Reply:
I think the comment is there as we still want to make it user-configurable.